tools/chef/image_sync.yaml

```diff
@@ -0,0 +1,36 @@
+---
+version: 1
+creds:
+- registry: gcr.io
+- registry: ghcr.io
+  user: "{{env \"GHCR_USER\"}}"
```

Comment:
Does regsync get these from the docker config?

Reply:
If you are already auth-ed but these are not set, it still works.

Perhaps I should remove them from this config but I imagine it will exist for other sync configs that might be run via gh-actions.